src/trip.core.js

```diff
@@ -507,7 +507,7 @@ Trip.prototype = {
     this.setTripBlock(o);
     this.showTripBlock(o);
 
-    if (this.settings.enableAnimation) {
+    if (TripUtils.isSet(o.enableAnimation, this.settings.enableAnimation)) {
```

Comment:
BTW, I am still thinking whether there is a better naming for `TripUtils.isSet` ?  hmmmm